stories/components/SummaryDetail.stories.svelte

```diff
@@ -1,45 +1,34 @@
 <script lang="ts">
-  import {Meta, Story} from '@storybook/addon-svelte-csf'
-  import FileUploadButton from '../../src/svelte/FileUploadButton.svelte'
+  import {Meta, Story} from '@storybook/addon-svelte-csf';
+  import SummaryDetail from '../../src/svelte/SummaryDetail.svelte';
 </script>
 
 <Meta
   title="Components/Summary Detail"
   args={{
-    disabled: false,
-    disableJs: false,
-    disableCss: false
+    title:'Eksempel på summary details',
   }}
-  argTypes={{
-    disabled: {control: 'boolean'},
-    disableJs: {control: 'boolean'},
-    disableCss: {control: 'boolean'}
-  }} />
+/>
 
-<Story name="Normal" let:primary let:primaryLong let:disableCss let:disabled let:secondary>
+<Story name="Normal" let:args >
   <main class="container layout-flex-col">
     <h1>Detail summary</h1>
```

Comment:
```suggestion
    <h1>Oppsummering og detaljer</h1>
```
Forslag til norsk navn?